mesheryctl/internal/cli/root/config/config.go

```diff
@@ -60,11 +62,12 @@ func (mc *MesheryCtlConfig) CheckIfCurrentContextIsValid() (Context, error) {
 			ctx.Version = "latest"
 		}
```

Comment:
you can also move this assignment of the version to "latest" if it's empty to that member function of `Context`